packages/flame_tiled/lib/src/renderable_layers/tile_layers/hexagonal_tile_layer.dart

```diff
@@ -119,6 +119,9 @@ class HexagonalTileLayer extends FlameTileLayer {
           offsetY = ty * size.y + staggerY + halfDestinationTile.y;
         }
 
+        offsetX += (tileset.tileOffset?.x ?? 0) * -1;
```

Comment:
Interesting, I would have thought it should be the y-offset that should be negated?

Reply:
I'll create test cases later for that and resubmit my PR.

Reply:
You don't have to re-submit the PR, you can just continue to push towards your already existing branch and the PR will update. :)

Reply:
That's what I meant ;-)